src/main/java/org/dcsa/ovs/service/impl/OVSEventSubscriptionTOServiceImpl.java

```diff
@@ -0,0 +1,281 @@
+package org.dcsa.ovs.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.dcsa.core.events.model.EventSubscription;
+import org.dcsa.core.events.model.base.AbstractEventSubscription;
+import org.dcsa.core.events.model.enums.*;
+import org.dcsa.core.events.repository.EventSubscriptionRepository;
+import org.dcsa.core.events.service.EventSubscriptionService;
+import org.dcsa.core.events.service.impl.EventSubscriptionTOServiceImpl;
+import org.dcsa.core.exception.UpdateException;
+import org.dcsa.core.util.MappingUtils;
+import org.dcsa.ovs.model.transferobjects.OVSEventSubscriptionTO;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
```

Comment:
The class says `Mainly for internal use within the framework; consider Apache's Common Lang for ...`

Reply:
I looked into , you're right. Though the implementation is a simple one , its better to use common lang. However not required anymore , so removed it. Will keep in mind for when I need to use it next time.